add LUKSO web3-onboard documentation (#617)
lukso-network/docs: docs/guides/browser-extension/web3-onboard.md

# Connect users' wallets to your Dapp with web3-onboard

> ### The easiest way to connect a wallet 🚀

Reviewer: Why do we go from h1 to h3 directly?

![web3-onboard-view](/img/extension/web3-onboard.png)

> Web3-onboard configuration allows the detection of the Universal Profiles (UP) Extension, along with any other installed browser extensions. Users can integrate it into their Dapp to handle the routing for the different browser extensions.

Reviewer 2: ```suggestion
Web3-onboard configuration allows the detection of the Universal Profiles (UP) Extension, along with any other installed browser extensions. Developers can integrate it into their Dapp to handle the routing for the different browser extensions.
```

> Web3-onboard is an open-source, framework-agnostic JavaScript library to onboard users to web3 apps. This package can be used to integrate LUKSO [Universal Profiles Extension](https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn?hl=en) support into web3-onboard's "Connect Wallet" modal. With this module, **the LUKSO Universal Profile Extension option will be shown even if the extension is not installed on the user's browser or used within the LUKSO Universal Profile Extension**. If selected the user will be taken to a download screen and prompted to create a LUKSO Universal Profile.

Reviewer 2: ```suggestion
Web3-onboard is an open-source, framework-agnostic JavaScript library to onboard users to web3 apps. This package can be used to integrate LUKSO [Universal Profiles Extension](https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn?hl=en) support into web3-onboard's "Connect Wallet" modal. With this module, **the LUKSO Universal Profile Extension option will be shown even if the extension is not installed on the user's browser**. If selected the user will be taken to a download screen where they can install the extension.
```
